nextflow_schema.json

```diff
@@ -520,6 +520,19 @@
                     "description": "Institutional config URL link.",
                     "hidden": true,
                     "fa_icon": "fas fa-users-cog"
+                },
+                "clusterOptions": {
+                    "type": "string"
+                },
```

Comment:
I think this one you can also delete from everywhere - it's a bit of a relic. I found it in the methylseq pipeline a few days ago also.